### PR TITLE
Fix screenbuilder help `Related Commands`

### PR DESCRIPTION
**Pull request by Mitko-Kerezov**
Ping @ikoevska

### CODE REVIEW

File: docs/man_pages/screenbuilder/screenbuilder.md [hunk 1]
```diff
@@ -28,4 +28,11 @@ Command | Description
 Command | Description
 ----------|----------
 [create screenbuilder](../project/creation/create-screenbuilder.html) | Creates a new project for hybrid development with Screen Builder.
+[add-about](add-about.html) | Inserts an about form in an existing application view.
```
Comment (ikoevska) [inline docs/man_pages/screenbuilder/screenbuilder.md]: This seems redundant here =) After all we have the exact same list a few lines above.

Comment (Mitko-Kerezov) [inline docs/man_pages/screenbuilder/screenbuilder.md]: Point taken

**Comment by Mitko-Kerezov** [comment]
Comment by @ikoevska addressed

**Comment by Mitko-Kerezov** [comment]
Closing this due to retarget towards release

**Pull request closed without merging**